api/v1/orchestratorscan/service.go

```diff
@@ -27,20 +28,20 @@ type Service interface {
 }
 
 // NewService returns the service for scanning
-func NewService(db database.Datastore, k8sCache k8scache.Cache) Service {
+func NewService(db database.Datastore, k8sCache k8scache.Cache, istioCache istioCache.Cache) Service {
 	return &serviceImpl{
-		version:  version.Version,
-		db:       db,
-		k8sCache: k8sCache,
+		version:    version.Version,
+		db:         db,
+		k8sCache:   k8sCache,
+		istioCache: istioCache,
 	}
 }
 
 type serviceImpl struct {
-	v1.UnimplementedOrchestratorScanServiceServer
```

Comment:
oops I missed this. Please re-add this, as it is useful for forwards-compatibility